modules/integration_azure_firewall/conf/01-snat.yaml

```diff
@@ -0,0 +1,21 @@
+## Example
+module: Azure firewall
+name: "SNAT Port Utilization"
+
+transformation: true
```

Comment:
it seems very dangerous for me to not set a little timeframe on cloud metrics which are sparsed by nature and have a big interval